src/components/icons/index.tsx

```diff
@@ -17,6 +17,7 @@ function RotatedIconComponent (
 }
 
 // simple
+export { default as Add } from './add'
```

Comment:
Maybe this should be plus instead of add, also it needs Icon at the end

Reply:
Fixed